core/lib/devise/strategies/remote_authenticatable.rb

```diff
@@ -1,19 +1,20 @@
 module Devise
   module Strategies
     class RemoteAuthenticatable < Authenticatable
-      
+
       # def valid?
       #   true || params[scope]
       # end
-      
+
       # For an example check : https://github.com/plataformatec/devise/blob/master/lib/devise/strategies/database_authenticatable.rb
       # Method called by warden to authenticate a resource.
       def authenticate!
         # authentication_hash doesn't include the password
         auth_params = params[scope]
-        
+
+
         # mapping.to is a wrapper over the resource model
-        resource = mapping.to.new
+        resource = mapping.to.authenticate(auth_params)
```

Comment:
I'd keep `mapping.to.remote_authenticate(auth_params)`

Reply:
you meant `mapping.to.new.remote_authentication(auth_params)` ?